android/cli/commands/_buildModule.js

```diff
@@ -221,69 +227,18 @@ AndroidModuleBuilder.prototype.validate = function validate(logger, config, cli)
 
 			// check the Android SDK we require to build exists
 			this.androidCompileSDK = targetSDKMap[this.compileSdkVersion];
-			if (!this.androidCompileSDK) {
-				logger.error(__('Unable to find Android SDK API %s', this.compileSdkVersion));
-				logger.error(__('Android SDK API %s is required to build Android modules', this.compileSdkVersion) + '\n');
-				process.exit(1);
-			}
 
 			// if no target sdk, then default to most recent supported/installed
 			if (!this.targetSDK) {
-				const levels = Object.keys(targetSDKMap).sort();
-
-				for (let i = levels.length - 1; i >= 0; i--) {
-					if (levels[i] >= this.minSupportedApiLevel && levels[i] <= this.maxSupportedApiLevel) {
-						this.targetSDK = levels[i];
-						break;
-					}
-				}
-
-				if (!this.targetSDK) {
-					logger.error(__('Unable to find a suitable installed Android SDK that is >=%s and <=%s', this.minSupportedApiLevel, this.maxSupportedApiLevel) + '\n');
-					process.exit(1);
-				}
+				this.targetSDK = this.maxSupportedApiLevel;
+			
```

Comment:
<!--
  1 failure:  android/cli/comma...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: android/cli/commands/_buildModule.js;
  Line: 234;
-->
- :no_entry_sign: android/cli/commands/_buildModule.js line 234 – Trailing spaces not allowed. (no-trailing-spaces)